### PR TITLE
Git prune plugin

### PR DESCRIPTION
**Pull request by Seinh**
I added a new plugin to remove unnecessary branches that are already merged in a repository, 
I hope you like it

Regards

### CODE REVIEW

**Comment by esteban-uo** [comment]
+1

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ custom
 *.swp
 !custom/example.zshcache
 cache/
+.DS_Store
```
Comment (RobLoach) [inline .gitignore]: Likely don't need this change in this Pull Request.

File: plugins/git-prune/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Git prune plugin
+This plugin allows you to delete all branches that are already merged in your local repository or/and that were merged in your remote repository (/origin)
```
Comment (RobLoach) [inline plugins/git-prune/README.md]: Well done on the docs here :+1: .

**Comment by Seinh** [comment]
I have removed the .gitignore changes, thank you for the feedback

**Comment by annalisavarrone** [comment]
+1

**Comment by Seinh** [comment]
@RobLoach I have made mayor improvements to my plugins

**Comment by norberto5757** [comment]
look with deep sincerity do not know when or I receive your e'm just a
systems programmer with a very broad curriculum right through 37 years of
living today than my children offer me forgiveness if at any time
incosiente were taxed. my data but not. nothing to disturb my social
environment much less in a country that I admire ye visited for several
years my name is norberto gonzalez rojas and my mobile in mexico is
6645872021 my address from my home in Tijuana, Baja californa is Andador A
#, 31 elRubi apologies please delete my information from your database ever
and hurt anyone thanks blessings also I inform you that do not speak
English use the Google translator only thing that 100% are programming
terms what I do on social networks is to speak God does not send me anyone
is not charging its own initiative or annoying people tendencista'm not
religious and I am apolotitico only love God above all things blessings.
Is las conversion you needed talk Hispanic me number+526643197230

**Pull request closed without merging**